examples/converters.py

```diff
@@ -0,0 +1,95 @@
+# This example requires the 'members' privileged intent to use the Member converter.
+
+import typing
+
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot("!", intents=intents)
+
+
+@bot.command()
+async def userinfo(ctx: commands.Context, user: discord.User):
+    # In the command signature above, you can see that the `user`
```

Comment:
This should maybe mention that type hinting means converting **in discord.py** since I have seen some users assume this means it happens in normal Python